client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -8,29 +8,40 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { shuffle } from 'lodash';
 import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
+import { localize, moment } from 'i18n-calypso';
```

Comment:
`localized` components will have moment, I don't believe we want to import it from here.

https://github.com/Automattic/wp-calypso/tree/master/packages/i18n-calypso#localize